lib/plugins/network/flipper_http_client_request.dart

```diff
@@ -131,27 +132,36 @@ class FlipperHttpClientRequest implements HttpClientRequest {
     request.writeln(obj);
   }
 
-  Future<FlipperHttpClientResponse> withInterceptor(HttpClientResponse response) async {
-    FlipperHttpClientResponse responseWithInterceptor = 
-      new FlipperHttpClientResponse(this.uniqueId, response);
+  @override
+  void abort([Object exception, StackTrace stackTrace]) =>
+      request.abort(exception, stackTrace);
```

Comment:
This is the only part where I actually changed anything my code formatter did the rest. Can revert if you prefer